tor_ocr/main.py

```diff
@@ -144,8 +145,7 @@ def decode_image_from_url(url, overlay=False, api_key=__OCR_API_KEY__):
 
     for API in __OCR_API_URLS__:
         try:
-            logging.info(f'Calling ocr.space API: {API}')
-            result = requests.post(API, data=payload)
+            result = requests.post(API, data=payload, timeout=2)
```

Comment:
Just looking to call out that the `timeout=2` part is measured in terms of seconds, and not 2 seconds for the whole request. It's 2 seconds to wait starting from when the request is fired off until the first bit is received in response. If that part takes longer than 2 seconds, we throw a `ConnectTimeout` exception.

LGTM! 👍 

<img width="733" alt="Requests library docs about timeout, explaining same concept as above" src="https://user-images.githubusercontent.com/1276113/50234232-3073b800-0383-11e9-8d02-96a838fe5e67.png">

Reply:
I added some inline documentation for this. 👍